WeatherTests/Modules/WeatherLocation/WeatherLocationDefaultPresenterSpec.swift

```diff
@@ -25,15 +25,15 @@ class WeatherLocationDefaultPresenterSpec: QuickSpec {
         
         context("When viewIsReady is called") {
```

Comment:
We should update the text on the tests, because the method names have changed. I knew it was a bad idea to that in that way, my bad.